### ISSUE TITLE
Select more than one storage page

### ISSUE THREAD

**Issue by resta3000** [undkonsorten/typo3-cute-mailing-ttaddress#1]
If there is more than one store page, the newsletter must be sent several times because the different pages cannot be merged into one recipient list.

**Comment by kanow** [comment]
Yes, for the moment only one sys_folder can be used. For now it's not planned to change this behaviour. Maybe in future.

**Comment by kanow** [comment]
Today we talked about that feature and rejected it. Multiple newsletters must be used for this. If anyone really still needs this feature, it will need to be funded.
